api/src/test/java/org/openmrs/module/cohort/api/dao/CohortGenericDaoTest.java

```diff
@@ -147,4 +149,11 @@ public void getByUuid_shouldReturnNullForVoidedOrRetiredCohort() {
 		
 		assertThat(dao.get(COHORT_UUID), nullValue());
 	}
+	
+	@Test
+	public void shouldVoidCohortM() {
+		CohortM cohortToVoid = dao.get(COHORT_UUID);
```

Comment:
Can you assert that the cohort is not voided before calling service.voidCohort?

Reply:
@dkayiwa i updated test to assert that the cohort is not voided before calling to service.voidCohortM.